addon/components/o-s-s/phone-number-input.ts

```diff
@@ -107,4 +110,14 @@ export default class OSSPhoneNumberInput extends Component<OSSPhoneNumberInputAr
     this.countrySelectorShown = false;
     this.filteredCountries = this._countries;
   }
+
+  @action
+  focusInput(): void {
+    this.inputElement?.focus();
+  }
+
+  @action
+  registerInputElement(el: HTMLElement): void {
+    this.inputElement = el;
+  }
```

Comment:
Didn't comment about this last time, but why do we need to manually focus ? 🤔

Reply:
The `input` doesn't fill the whole component clicking on the +33 here does nothing without this manual focus
![Capture d’écran 2023-11-28 à 17 05 20](https://github.com/upfluence/oss-components/assets/111493996/9f41cf9a-614f-4d9c-84ab-a5b60a4469c3)